packages/backend/src/studio-api-impl.spec.ts

```diff
@@ -176,3 +184,25 @@ test('if requestDeleteLocalRepository fails an errorMessage should show up', asy
   expect(deleteLocalRepositoryMock).toHaveBeenCalled();
   expect(errorMessageMock).toBeCalledWith('Error deleting local path "path". Error: error deleting');
 });
+
+test('check openVSCode generates the correct URL', async () => {
+  const folder = path.resolve('.');
+
+  mocks.uriFileMock.mockImplementation((path: string) => {
+    return {
+      path: path,
+      with: (change?: { scheme?: string; authority?: string; path?: string; query?: string; fragment?: string }) => {
+        return {
+          path: path,
+          ...change,
+        };
+      },
+    };
+  });
+
+  mocks.openExternalMock.mockResolvedValue(true);
+  await studioApiImpl.openVSCode(folder);
+  expect(mocks.openExternalMock).toHaveBeenCalledWith(
+    expect.objectContaining({ path: expect.stringMatching(/^\//), authority: 'file', scheme: 'vscode' }),
+  );
+});
```

Comment:
for CI there is no pb as it's running both on Windows and macOS (so path.resolve will work)

but usually unit tests cover both Windows and macOS/Linux cases by testing the two ways

so you can see problems before CI running the test locally either on Windows or on macOS/Linux

like using a test with multiple argument('one with a path obtained from windows) and one from macOS/Linux)
`test.each([...`